extensions/configuration-editing/package.json

```diff
@@ -11,7 +11,9 @@
   "icon": "images/icon.png",
   "activationEvents": [
     "onProfile",
-    "onProfile:github"
+    "onProfile:github",
+    "onLanguage:json",
```

Comment:
I feel like looking for `settings.json` may be a bit narrower in the activation events

Reply:
I don't think there's a way to activate when a certain file is opened?

Reply:
ahh sadly not.